alvr/dashboard/src/dashboard/components/settings.rs

```diff
@@ -160,31 +160,25 @@ impl SettingsTab {
                         .striped(true)
                         .num_columns(2)
                         .show(ui, |ui| {
-                            ui.add_space(INDENTATION_STEP);
```

Comment:
i remember this being pretty important... did something change with updates?

Reply:
Yeah. The new update forbids the use of advance cursor inside a grid. `add_space` uses `advance_cursor` inside.. I've moved the call in the preset control itself inside a `horizontal` layout. The previous code was proken anyway because it defined 2 columns but 3 elements per row were added.